Annotation/constituent_golden_set.json

```diff
@@ -266,7 +273,7 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "how messed up things were for her",

```

Comment:
This is a degree clause, maybe not interrogative? So maybe not what we're looking for

Reply:
I'm not sure if we should just remove this one then?